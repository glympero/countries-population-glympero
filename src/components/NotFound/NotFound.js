import React from "react";

const NotFound = () => (
  <div>
    <h1>This is 404 page</h1>
  </div>
);

export default NotFound;
