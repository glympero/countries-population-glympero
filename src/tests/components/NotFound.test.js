import React from "react";
import { shallow } from "enzyme";
import NotFound from "../../components/NotFound/NotFound";

test("should render Not Found correctly", () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper.find("h1").length).toBe(1);
  expect(wrapper.find("h1").text()).toBe("This is 404 page");

  expect(wrapper).toMatchSnapshot();
});
