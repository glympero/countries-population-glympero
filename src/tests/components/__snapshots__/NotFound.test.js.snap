// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Not Found correctly 1`] = `ShallowWrapper {}`;
